sysconfig/bb5/users/packages.yaml

```diff
@@ -127,10 +123,6 @@ packages:
         paths:
             pango@1.40.4: /usr
         version: [1.40.4]
-    pcre:
-        paths:
-            pcre@8.32: /usr
-        version: [8.32]
     perl:
```

Comment:
I assume you might have seen error like `==> Error: An unsatisfiable variant constraint has been detected for spec:`. This is because `git` ask for `depends_on('pcre+jit'...')` and our spec is not sufficient enough. Instead of removing package (which is installed on system), it's sufficient to do:

```
-            pcre@8.32: /usr
+            pcre@8.32+jit+utf: /usr
```

this I have just tested and it works fine.